DO NOT SQUASH: Implement initial support for CQL-related operations (#2577)
IBM/FHIR: fhir-cql-rest/README.md

#fhir-cql-rest

> This module provides an implementation of data and terminology providers for the open source CQL engine using an IBM-FHIR REST client. This backend can be used by clients that want to run outside of the IBM FHIR server, but without the HAPI-FHIR dependencies that come with the open source CQL Engine's FHIR backend.

Reviewer: ```suggestion
This module provides an implementation of data and terminology providers for the open source CQL engine using an IBM FHIR Server REST client. This backend can be used by clients that want to run outside of the IBM FHIR Server.  There are no HAPI-FHIR dependencies used with the CQL Engine's FHIR backend.
```
